webui/src/components/Dashboard/Dashboard.vue

```diff
@@ -0,0 +1,142 @@
+<!-- Copyright (c) 2024 Seagate Technology LLC and/or its Affiliates -->
+<template>
+  <v-container style="width: 100%; height: 80vh">
+    <h2 style="text-align: center; margin-bottom: 20px">
+      Devices Connection Status
```

Comment:
Alt title suggestion

```suggestion
      CFM Ethernet Connections
```

then, for cxl, you could use `CFM CXL Connections`